test/logstream/kubelogs.go

```diff
@@ -77,7 +77,13 @@ func (k *kubelogs) startForPod(eg *errgroup.Group, pod *corev1.Pod) {
 			// Read this container's stream.
 			scanner := bufio.NewScanner(stream)
 			for scanner.Scan() {
-				k.handleLine(scanner.Bytes(), pn)
+				// Specialcase logs from chaosduck to be able to easily see when pods
+				// have been killed throughout all tests.
+				if cn == "chaosduck" {
+					k.handleGenericLine(scanner.Bytes(), pn)
+				} else {
+					k.handleLine(scanner.Bytes(), pn)
+				}
```

Comment:
given the amount of logs processing we're doing — may be just pass the processor func to the function
and check for the type of binary once?